samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -107,32 +107,92 @@ static float sqrt(float square)
 	return root;
 }
 
+static s32_t ceiling(float num)
+{
+	s32_t inum;
+
+	inum = (s32_t) num;
+	if (num == (float) inum) {
+		return inum;
+	}
+
+	return inum + 1;
+}
+
+static bool constrain_lightness_actual_state(u16_t var)
+{
+	bool is_value_within_range;
+
+	is_value_within_range = false;
+
+	if (var > 0 && var <
+		light_lightness_srv_user_data.lightness_range_min) {
```

Comment:
This is incorrectly aligned which makes it very confusing to look at, especially since the var assignment starts with the same alignment level. The split should happen after `&&` and the line should begin right after the `(` on the first line.

Reply:
done !!